src/registrar/index.js

```diff
@@ -1,152 +0,0 @@
-var _ = require('lodash'),
-	DOMAIN_SETTINGS_KEY = 'DOMAIN_SETTING_KEY',
```

Comment:
it's used on line 65.  it holds things like `name` and `label` that are passed in when doing something like the below in an app.
```js
var domain = app.createDomain({name: 'focal-sample', label: 'Focal Sample'});
```

these properties are stored by the registrar and become available within the framework when doing something like
```js
var domain = app.getDomain('focal-sample');
alert(domain.label);
```